stock_inventory_adjustment/views/stock_inventory.xml

```diff
@@ -0,0 +1,124 @@
+<odoo>
+
+    <record id="view_inventory_group_form" model="ir.ui.view">
+        <field name="name">stock.inventory.form.view</field>
+        <field name="model">stock.inventory</field>
+        <field name="priority">1000</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <button
+                        type="object"
+                        name="action_state_to_in_progress"
+                        class="oe_highlight"
+                        attrs="{'invisible':[('state', 'in', ['1', '3'])]}"
+                        string="Begin Adjustments"
+                    />
+                    <button
+                        type="object"
+                        name="action_state_to_draft"
+                        attrs="{'invisible':['|',('state', 'in', ['2', '3']), ('count_stock_moves', '!=', 0)]}"
+                        string="Back to Draft"
+                    />
+                    <button
+                        type="object"
+                        name="action_state_to_done"
+                        class="oe_highlight"
+                        attrs="{'invisible':[('state', 'in', ['2', '3'])]}"
+                        string="Set to Done"
+                    />
+                    <field name="state" widget="statusbar" statusbar_visible="2,1,3" />
+                </header>
+
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button
+                            type="object"
+                            name="action_view_inventory_adjustment"
+                            class="oe_stat_button"
+                            icon="fa-pencil-square-o"
+                            string="Adjustments"
+                            attrs="{'invisible':[('state', 'in', ['2', '3'])]}"
+                        >
+                            <field name="count_stock_quants" readonly="1" />
```

Comment:
```suggestion
                            attrs="{'invisible':['|', ('state', 'in', ['2', '3']), ('count_stock_quants', '=', 0)]}"
                        >
                            <field name="count_stock_quants" readonly="1" string="Adjustments" />
```
For all smart buttons:
- Move the string to the field
- Also, make it invisible if the count is 0.

Reply:
Done!

Reply:
You changes are done :smile: